README.md

```diff
@@ -4,127 +4,132 @@
 
 ## Introduction
 
-This helm chart installs the [Scalyr Agent monitor](https://app.scalyr.com/help/scalyr-agent) into a Kubernetes
-cluster. It supports installing the agent with all features required to support a Kubernetes cluster monitoring.
-Additionally, it can deploy Scalyr agents which monitor other parts of the infrastructure (for example a hosted
-database service etc.).
+This helm chart installs the [Scalyr Agent](https://app.scalyr.com/help/scalyr-agent) in a Kubernetes
+cluster. Two Agent Plugins plugins are also installed:
+- The [Kubernetes monitor](https://app.scalyr.com/monitors/kubernetes) enables the Agent to monitor Kubernetes clusters.
```

Comment:
I think it would be better to say something along the lines of two monitors / plugins are enabled. Since those monitors are also installed by default with other installations, they are just not enabled.

And perhaps, to be consistent and avoid confusion we should refer to it as "monitors" and not "plugins since afak, that's what we use in most places.

Reply:
And perhaps we also want to add a quick note and link here to the Kubernetes Explorer section - something along the lines - to gather more insight and access to additional Kubernetes cluster and application metrics you can enable Kubernetes Explorer (in preview phase), see section below on more info on how to enable it.

Reply:
Ah, OK, in case plugins is used in docs already then it should be fine.

And as you pointed out this may help with the distinction between agent and "remote" / server side (e.g. http, okta, etc.) monitors.